editor/export_template_manager.cpp

```diff
@@ -314,7 +314,8 @@ bool ExportTemplateManager::_install_from_file(const String &p_file, bool p_use_
 		if (!f) {
 			ret = unzGoToNextFile(pkg);
 			fc++;
-			ERR_CONTINUE(!f);
+			ERR_EXPLAIN("Can't open file from path: " + String(to_write));
+			continue;
```

Comment:
Should be `ERR_CONTINUE();`, otherwise the explanation won't be shown and no error will appear (`ERR_EXPLAIN()` only sets the error message for the next `ERR_FAIL/CONTINUE/BREAK` call).

Reply:
Well can't be `ERR_CONTINUE` without condition, which I guess is why you changed it. Should be `ERR_CONTINUE(true)` then.